webapp/webapp/webapp/lib/load_config.py

```diff
@@ -75,12 +75,17 @@ def wait_for_db(self, app):
                 print('   '+str(len(tables))+' populated')
                 time.sleep(2)
         sysconfig = r.db('isard').table('config').get(1).run(conn)
-        if 'shares' in sysconfig.keys():
+        app.shares_templates = sysconfig.get('shares', {}).get('templates', False)
+        app.shares_isos = sysconfig.get('shares', {}).get('isos', False)
+
+        print(app.shares_templates)
+        print(app.shares_isos)
```

Comment:
Are the prints required?

Reply:
uh, my fault! Just pushed without. Thanks!